.travis.yml

```diff
@@ -16,4 +16,4 @@ script:
   - echo 'paths = ["."]' > .cargo/config
   - git clone -q --depth 50 -b pending https://github.com/rust-gnome/examples _examples
```

Comment:
This IMHO need be `github.com/gtk-rs/examples` and on master branch,
but seem it need be done after merge or at minimum after merge "crate-release-update into master"

Reply:
Oh wow. That's quite the old URL!

Reply:
BTW, we need to figure out what's happening with cairo before moving forward on the release (for live time info, I recommend coming on IRC ;) ).

Reply:
@GuillaumeGomez Which IRC? You seems not on #gtk-rs on irc.mozilla.org

Reply:
Answered on the IRC. :)